tests/data/swc/section_order.swc

```diff
@@ -0,0 +1,37 @@
+1 1 0 0 0 10 -1
+
+# section 11
+2 4 0 0 1 0.5 -1
```

Comment:
1?

Reply:
You mean the section ID should be 1. Nope, this section is an apical dendrite so it goes last.

Reply:
Sorry, I forgot that it's Gerrit that highlights what I was looking at.  I mean the parent ID should 1, since I assume it's connected to the soma

Reply:
Yes, and no. The implementation is resilient to first order branches not being connected to anything. I didn't connect this one to the soma on purpose.

(Despite all harsh criticism that gerrit gets for it's UI, I still think that code reviews are more effective there)

Reply:
Ok, fair enough.  Maybe leave a comment so the reasoning is captured.

Agree about the gerrit review